### PR TITLE
draco: fixes draco3d memory leak

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/draco/src/lib/draco-module-loader.js [hunk 1]
```diff
@@ -6,14 +6,17 @@ import {loadLibrary, global} from '@loaders.gl/loader-utils';
 
 let loadDecoderPromise;
```
Comment (ibgreen) [inline modules/draco/src/lib/draco-module-loader.js:7]: Now that I look at this again, I am wondering why these promises didn't handle the deduplication?

If loading already started, then the same promise should always be returned...

Comment (belom88) [reply modules/draco/src/lib/draco-module-loader.js:7]: As you can see, `loadDecoderPromise` is never assigned when we have 'draco3d' module in options. In 'draco3d' npm package there are outstanding `createDecoderModule` and `createEncoderModule`. These are not promises but functions that return ready for usage a draco module.

Comment (ibgreen) [reply modules/draco/src/lib/draco-module-loader.js:7]: Would this work?
```js
if (modules.draco3d) {
  loadDecoderPromise = Promise.resolve(modules.draco3d.createDecoderModule());
}
```

Comment (belom88) [reply modules/draco/src/lib/draco-module-loader.js:7]: Possibly. I'll try it.

Comment (belom88) [reply modules/draco/src/lib/draco-module-loader.js]: Done

**Review by ibgreen — APPROVED** [review]

**Pull request merged**